### PR TITLE
[risk=low][RW-13861] Config fixes to run v8 extraction in test

### PR DESCRIPTION
**Pull request by jmthibault79**
---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: api/config/config_local.json [hunk 1]
```diff
@@ -55,6 +55,12 @@
       "methodLogicalVersion": 3,
       "minExtractionScatterTasks": 20,
       "extractionScatterTasksPerSample": 4
+    },
+    "cdrv8plus": {
+      "methodNamespace": "aouwgscohortextraction-test",
```
Comment (jmthibault79) [inline api/config/config_local.json:60]: reflects the most recent version tested in #8901

File: api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java [hunk 2]
```diff
@@ -364,11 +385,8 @@ private Map<String, String> getWorkflowInputs(
                 + personIdsFile.getName()
                 + "\"")
         .put(EXTRACT_WORKFLOW_NAME + ".query_project", "\"" + workspace.getGoogleProject() + "\"")
-        // Added in https://github.com/broadinstitute/gatk/pull/7698
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java:367]: these fields are version-dependent, as determined from testing in #8901

File: api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java [hunk 3]
```diff
@@ -340,9 +340,30 @@ private Map<String, String> getWorkflowInputs(
       maybeInputs.put(EXTRACT_WORKFLOW_NAME + ".filter_set_name", "\"" + filterSetName + "\"");
     }
 
-    scatterCount.ifPresent(
-        count ->
-            maybeInputs.put(EXTRACT_WORKFLOW_NAME + ".scatter_count", Integer.toString(count)));
+    if (useLegacyWorkflow) {
+      // Initial heuristic for scatter count, optimizing to avoid large compute/output shards while
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java:344]: moved up from L420 to group with newer logic dependent on useLegacyWorkflow

File: api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java [hunk 4]
```diff
@@ -340,9 +340,30 @@ private Map<String, String> getWorkflowInputs(
...
+      int desiredScatter =
+          Math.round(
+              personIds.size()
+                  * cohortExtractionConfig.legacyVersions.extractionScatterTasksPerSample);
+      int scatterCount =
+          Ints.constrainToRange(desiredScatter, minScatter, LEGACY_MAX_EXTRACTION_SCATTER);
+
+      maybeInputs.put(EXTRACT_WORKFLOW_NAME + ".scatter_count", Integer.toString(scatterCount));
+
+      // Added in https://github.com/broadinstitute/gatk/pull/7698
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java:359]: moved from the universal inputs section.  v8 doesn't have these.

**Review by Qi77Qi — APPROVED** [review]
let's give it a try

**Pull request merged**